docs/general/server/media/video.md

```diff
@@ -0,0 +1,338 @@
+   - With `Airs after series`, set the Special will be shown (and played) at the end of the specified Season. This will take priority over the `Airs before season/episode` fields if they are set.
+   - When multiple specials have the same position, they will play in the order in which they are stored in the specials season. For example, if S00E01 and S00E03 both have `Airs before season` set to `3` and `Airs before episode` set to `7`, the order the content will play in is: S03E06 → S00E01 → S00E03 → S03E07.
+
+Note that this will show them in both the `Specials` season, in addition to the season specified.
+
+## Music Videos
+
+Music videos are organized mostly the same as movies, with a few key differences - no external metadata providers are available by default, and multi layer folders are supported.
+
+```txt
+MusicVideos
+├── Folder 1
+│   ├── Folder 1.1
+│   │   └── Song 1
+│   │       ├── Song 1 Version 1.mp4
+│   │       ├── Song 1 Version 2.mp4
+│   │       └── Song 1 Version 3.mp4
+│   ├── Song 2
+│   │   ├── Song 2 Version 1.mp4
+│   │   ├── Song 2 Version 1.nfo
+│   │   ├── Song 2 Version 2.mp4
+│   │   └── Song 2 Version 2.nfo
+│   └── Song 3
+│       ├── Song 3.mkv
+│       └── Song 3.nfo
+└── Song 4
+    └── Song 4.avi
+```
+
+## File Suffixes
+
+### External Subtitles and Audio Tracks
+
+External Subtitles can be added with file suffixes.
+
+```txt
+/Movies
+    /Film (1986)
+        Film.mkv
+        Film.default.srt
+        Film.default.en.forced.ass
+        Film.forced.en.dts
+        Film.en.sdh.srt
+        Film.English Commentary.en.mp3
+```
+
+Each title/flag field can be a generic string, or a special flag. A file can have multiple, separated with `.`
+
+| Type             | Flag                |
+| ---------------- | ------------------- |
+| Default          | `default`           |
+| Forced           | `forced`, `foreign` |
+| Hearing Impaired | `sdh`, `cc`, `hi`   |
+
+`hi` collides with the Hindi language abbreviation. `hi` by itself will resolve as a Hindi language track, while `hi` in conjunction with another language identifier (such as `title.en.hi.srt`) will use the other language and tag it as hearing impaired.
+
+Flags are ignored on containers with more than one stream.
+
+Any arbitrary text not parsable to a language or flag will be combined and used as the title of the stream (if there is not a stream title already embedded in the file metadata). The last file in the above example will be parsed as an English mp3 audio stream with the title `English Commentary`.
+
+### Multiple Versions
+
+Multiple versions of the same video can be stored together in the library using a file suffix. The part before any suffixes **MUST** be exactly the same, including any years and/or metadata provider IDs.
```

Comment:
Under which circumstances they are supported should be mentioned, otherwise people will assume series work.